packages/shared/src/components/utilities.tsx

```diff
@@ -70,10 +75,30 @@ const themeColors = {
     button: 'btn-primary',
   },
 };
+
 export const getThemeColor = (color: string, fallback: string): ThemeColor => {
   return themeColors[color] ?? themeColors[fallback];
 };
 
+const themeFonts = {
+  bodyBold: {
+    heading: 'font-bold typo-body',
```

Comment:
I actually think we can omit the heading from this, as there will not be any other versions.

Reply:
@rebelchris please have a look and resolve if it is ok.